core/modules/qdisp/QueryRequest.h

```diff
@@ -50,7 +51,7 @@ class BadResponseError : public std::exception {
     BadResponseError(std::string const& s_)
         : std::exception(),
           s("BadResponseError:" + s_) {}
-    ~BadResponseError() throw() {}
+    virtual ~BadResponseError() throw() {}
     virtual char const* what() throw() {
```

Comment:
This is one of the warnings that I picked up from clang in [DM-3803](https://jira.lsstcorp.org/browse/DM-3803).